conceptnet5/vectors/query.py

```diff
@@ -187,6 +196,12 @@ def expanded_vector(self, terms, limit_per_term=10, include_neighbors=True):
             self.expand_terms(terms, limit_per_term, include_neighbors)
         )
 
+    def get_fasttext_vector(self, query):
+        """Look up a vector for a query in the fastText model."""
+        label = query.split('/')[-1]
```

Comment:
Good point about not being sure about the last segment. I can see two approaches to deal with it:
1. reducing the uri with `uri_prefix` before looking it up. Perhaps adding `uri_prefix` to other methods as well.
2. keeping it consistent with other methods and not returning a vector.
I like 1. more.

Regarding the second point, `get_fasttext_vector()` is a method of `VectorSpaceWrapper`, which assumes a label is in a standard form.